src/js/utils.js

```diff
@@ -125,6 +125,31 @@ function secsToMins(secs, withSuffix = false) {
   }
 }
 
+/**
+ * Restraining input as integer from 1 to 99
+ * If input is a number but not with in 1~99, silently make it as close a possible.
+ * If input is within range but not an integer, silently round to nearest int.
+ *
+ * @param {Number} numInput numInput that is expected to be a number from 1 to 99.
```

Comment:
```suggestion
 * @param {String} numInput String representing a Number.
```
It looks like the input accepted here is actually a String, right? Since we have to call `parseFloat()` on it. (Ideally we wouldn't need to worry about this here, but I don't think we'll have time to fix this stuff before tomorrow)